lib/plivo/resources/messages.rb

```diff
@@ -61,60 +61,129 @@ def get(message_uuid)
       # @option options [String] :method The method used to call the url. Defaults to POST.
       # @option options [String] :log If set to false, the content of this message will not be logged on the Plivo infrastructure and the dst value will be masked (e.g., 141XXXXX528). Default is set to true.
       # @option options [String] :trackable set to false
-      def create(src, dst, text, options = nil, powerpack_uuid = nil)
-        valid_param?(:src, src, [Integer, String, Symbol], false)
-        valid_param?(:text, text, [String, Symbol], true)
-        valid_param?(:dst, dst, Array, true)
-        valid_param?(:powerpack_uuid, powerpack_uuid, [String, Symbol], false)
-        dst.each do |dst_num|
-          valid_param?(:dst_num, dst_num, [Integer, String, Symbol], true)
-        end
+      def create(src = nil,  dst = nil, text = nil, options = nil, powerpack_uuid = nil)
```

Comment:
why dst is nil ? In which case, do you think, its required ?

Reply:
Ruby isn't considering it as hash if I submit both kwargs & args in a method. However, added another check to ensure dst param is never nil.